src/mappings/utils.ts

```diff
@@ -101,10 +101,15 @@ export function dateToDayKey(timestamp: any): number {
 
 export const getEscrowAddress = (port: string, channel: string) => {
   const version = 'ics20-1';
-  const chainPrefix = 'agoric';
   const stringtoSha = Buffer.from([...Buffer.from(version), 0, ...Buffer.from(`${port}/${channel}`)]);
   const shaHash = sha256.sha256.array(stringtoSha.toString());
-  const bechWords = bech32.toWords(shaHash.slice(0, 20));
-  const address = bech32.encode(chainPrefix, bechWords);
+  const address = getAddressFromUint8Array(shaHash.slice(0, 20));
   return address;
 };
+
+export const getAddressFromUint8Array = (uint8Array: Array<number>, chainPrefix: string = 'agoric') => {
```

Comment:
You can use the `Uint8Array` type, 

```
export const getAddressFromUint8Array = (uint8Array: Uint8Array, chainPrefix: string = 'agoric') => {

```

Reply:
If you change the type, you will need to make this change on line 106:

```js
  const shaHashUint8Array = new Uint8Array(shaHash.slice(0, 20));
  const address = getAddressFromUint8Array(shaHashUint8Array);
```

Reply:
PS: I haven't tested it. Just checked in the code and there was no compilation errors.